doc/source/_templates/layout.html

```diff
@@ -1,6 +1,134 @@
 {% extends "!layout.html" %}
 
 {% block rootrellink %}
+<li>
+<span class="version_switcher_placeholder">{{ release }}</span>
+<script type="text/javascript">
+(function() {
+  'use strict';
+
+  var all_versions = {
+    'dev': 'dev',
+    '1.15.1': '1.15.1',
+    '1.14.5': '1.14.5',
+    '1.13.0': '1.13.0',
+    '1.12.0': '1.12.0',
+    '1.11.0': '1.11.0',
+    '1.10.1': '1.10.1',
+    '1.9.3': '1.9.3',
+    '1.8.1': '1.8.1',
+    '1.7.0': '1.7.0',
+  };
```

Comment:
Why a dictionary with identical keys and values?

Reply:
Inherited from [CPython Source](https://github.com/python/cpython/blob/master/Doc/tools/static/switchers.js#L12); might be able to simplify if it stays that way, but maybe not if we decide to have a special handling for `dev`